app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,7 @@
+class QuestionsController < ApplicationController
+  def question_content
+    @keyword = Keyword.find_by(translit: params[:keyword])
+    @questions =  @keyword.with_questions_and_users
+    @urls = Keyword.select_21_link_in_random_order
```

Comment:
This variable naming doesn't makes any sense.
```
@urls = Keyword.select_21_link_in_random_order
```
Like I said previously  "select_21_link_in_random_order" should be renamed to => in_random_order
that means variable @urls should be renamed to @random_keywords